packages/web-components/src/tooltip/tooltip.styles.ts

```diff
@@ -0,0 +1,95 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '../utils/display.js';
+import {
+  borderRadiusMedium,
+  colorNeutralBackground1,
+  colorNeutralForeground1,
+  colorNeutralShadowAmbient,
+  colorNeutralShadowKey,
+  fontFamilyBase,
+  fontSizeBase200,
+  lineHeightBase200,
+  spacingHorizontalXS,
+  spacingVerticalXS,
+} from '../theme/design-tokens.js';
+import { TooltipPositioning } from './tooltip.options.js';
+
+/**
+ * Styles for the tooltip component
+ * @public
+ */
+export const styles = css`
+  ${display('inline-flex')}
+
+  :host(:not(:popover-open)) {
+    display: none;
+  }
+
+  :host {
+    --blockOffset: ${spacingVerticalXS};
+    --inlineOffset: ${spacingHorizontalXS};
+    background: ${colorNeutralBackground1};
+    border-radius: ${borderRadiusMedium};
+    border: 1px solid var(--colorTransparentStroke);
+    box-sizing: border-box;
+    color: ${colorNeutralForeground1};
+    display: inline-flex;
+    filter: drop-shadow(0 0 2px ${colorNeutralShadowAmbient}) drop-shadow(0 4px 8px ${colorNeutralShadowKey});
+    font-family: ${fontFamilyBase};
+    font-size: ${fontSizeBase200};
+    inset-area: block-start;
+    line-height: ${lineHeightBase200};
+    margin: 0;
+    max-width: 240px;
+    padding: 4px 11px 6px;
```

Comment:
Does 11px come from deisgn? Or the react component? Seems out of place

Reply:
Comes from the React component. The `padding-block-*` values seem to be font-metric based but yea, not sure where 11px comes from in a token sense. If we wanted to tokenize we could move it up or down 1px.

```
mNudge: '10px',
m: '12px',
```

Reply:
Went with `spacingHorizontalMNudge` (10px)